crates/sui-indexer-alt-e2e-tests/tests/graphql/transactions/gas_input.snap

```diff
@@ -147,3 +151,31 @@ Response: {
     }
   }
 }
+
+task 12, lines 131-152:
+//# run-graphql
+Response: {
+  "data": {
+    "systemTransaction": {
+      "gasInput": {
+        "gasSponsor": {
+          "address": "0x0000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
You want this to return `null` here, which means `GasInput` needs to be updated to check for `0x0` and return `null` in this case.